pkg/network/mtu.go

```diff
@@ -0,0 +1,38 @@
+package network
+
+import (
+	"net"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+// GetDefaultMTU gets the mtu of the default route.
+func GetMTUTo(dest net.IP) (int, error) {
+	// Get the interface with the default route
+	routes, err := netlink.RouteGet(dest)
```

Comment:
If you actually want the interface with the default route, you can find that without needing a `dest`; just get the list of all routes, and find the default one. There is code to do this [in openshift](https://github.com/openshift/origin/blob/master/pkg/network/node/pod.go#L367).

In theory, someone could have routing set up such that the default route is not actually the route to the apiserver, but in theory, they could also have it set up that the route to the apiserver is not the route to the nodes. They can manually fill in the MTU in that case.

Reply:
Yeah, I was indeed thinking of that case, but you'r right - that's overthinking it. I'll switch it to the default route.